third_party/CMakeLists.txt

```diff
@@ -136,6 +136,7 @@ add_subdirectory(libarchive)
 target_include_directories(archive_static INTERFACE
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/libarchive/libarchive>)
 
+
```

Comment:
The addition of this blank line is an example of a superfluous change that you should remove.